amm/traits/pair.rs

```diff
@@ -82,6 +82,11 @@ pub trait Pair {
     /// Returns address of the second token.
     #[ink(message)]
     fn get_token_1(&self) -> AccountId;
+
+    /// Returns the trading fee of the pair.
+    /// Denominated in basis points (1/100 of a percent).
```

Comment:
Basis points are 10^{-4} and I think this outputs as 10^{-3}.

Reply:
Stupid ChatGPT :P I didn't validate every word.